test/test617.js

```diff
@@ -0,0 +1,33 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '617'; // insert test file number
+
+describe('Test ' + test + ' - Where LTRIM will remove leading whitespace characters in an expression.', function() {
+
+    it('A) Will remove leading whitespace only', function() {
+        var sql = "select LTRIM('      Hello World !')";
+        var res = alasql(sql);
+        assert.equal(res[0][Object.keys(res[0])[0]], 'Hello World !');
```

Comment:
This is had to read for new people coming in. I suggest using `AS ...` to indicate an exact name - or prepend `VALUE OF ` to the `SELECT` to get only the answer

for example: 

```
var sql = "select LTRIM('      Hello World !') AS str";
var res = alasql(sql);
assert.equal(res[0]['src'], 'Hello World !');
```

or something like 

```
var sql = "VALUE OF select LTRIM('      Hello World !')";
assert.equal(alasql(sql), 'Hello World !');
```